tests/data/tools/fail_doi.xml

```diff
@@ -0,0 +1,31 @@
+<tool id="copy" name="Copy Dataset" version="1.0">
+    <description>copies a dataset</description>
```

Comment:
The test framework just ensures all ``fail_`` tools fail linting - just rename to invaild_doi or something and the failing test will work.